HeroHoursRemake/settings.py

```diff
@@ -95,7 +95,7 @@
     'default': dj_database_url.config(default=os.environ['DATABASE_URL'], conn_max_age=600, ssl_require=True)
 }
 default_database = os.environ.get('DJANGO_DATABASE', 'default')
-#DATABASES['default'] = DATABASES[default_database]
+DATABASES['default'] = DATABASES[default_database]
```

Comment:
Ok feel free to make that in a separate pr. I just changed this because otherwise I needed to change my local setting.py everytime I checked out to test locally.